app/src/pages/trace/SpanDetails.tsx

```diff
@@ -287,6 +326,28 @@ export function SpanDetails({
   );
 }
 
+const spanInfoWrapCSS = css`
+  flex: 1 1 auto;
+  overflow-y: auto;
+  // Overflow fails to take into account padding
```

Comment:
could this be solved with box-sizing or something?

Reply:
I tried box-sizing and it's not working. Not sure why